aiocoap/numbers/codes.py

```diff
@@ -161,8 +161,46 @@ def _repr_html_(self):
         else:
             return f'<abbr title="Unknown {self._classification()}Code">{self.dotted}</abbr>'
 
-for k in vars(Code):
-    if isinstance(getattr(Code, k), Code):
-        locals()[k] = getattr(Code, k)
+EMPTY = Code.EMPTY
```

Comment:
Do you have any idea how we could do this in a way that doesn't lead to items silently being missing on this side when just added to Code?

Reply:
(Whatever we come up with here could also be applied to https://github.com/chrysn/aiocoap/pull/344)